ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -76,6 +76,8 @@
 
 public class BeaconStateUtil {
 
+  public static boolean BLS_VERIFY_DEPOSIT = true;
```

Comment:
Added this test https://github.com/PegaSysEng/artemis/pull/1089/commits/b7288de9f0908a181ebcf4b1e7a2a767442691f4
Did you mean something like this?

Reply:
That test is probably good to have too, but I was thinking of the really straight forward:

```
@Test
void ensureVerifyDepositDefaultsToFalse() {
  assertThat(BeaconStateUtil.BLS_VERIFY_DEPOSIT).isTrue();
}
```

Purely to ensure something fails if someone changes that to false for testing or something and accidentally commits it.

Reply:
Yep, I see what you mean. My test covers this case. 
Added your variant as well

Reply:
Ah yes it would too, sorry.  I know I'm very paranoid about "disable validation" flags like these so glad to have the very explicit boolean is true test as well.